lib/src/auth/frappe/frappe.auth.controller.dart

```diff
@@ -491,6 +492,34 @@ class FrappeAuthController extends AuthController<FrappeSessionStatusInfo> {
     }
   }
 
+  @override
+  Future<ResetPasswordInfo> getPasswordResetInfo({
```

Comment:
The return type for getPasswordResetInfo is different from the rest since normally we would do 
`RequestResponse<T>`

Reply:
Fixed

Reply:
Actually, since I brought this up, I think we should give a second look at all of the returns from controllers in general I believe. Since I noticed the response from auth.controller.dart includes `SendOTPResponse` & `VerifyOTPResponse`, to unify it at least.